x-pack/plugins/ml/common/constants/field_types.ts

```diff
@@ -9,6 +9,7 @@ export const ML_JOB_FIELD_TYPES = {
   BOOLEAN: 'boolean',
   DATE: 'date',
   GEO_POINT: 'geo_point',
+  GE0_SHAPE: 'geo_shape',
```

Comment:
Shouldn't this be `GEO_SHAPE` (`O` instead of `0`)?

Reply:
Thanks for catching that! Renamed here https://github.com/elastic/kibana/pull/92198/commits/96368a89397b1adebc681be9dfae08104e7af11c